plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -114,26 +113,25 @@ function perflab_render_plugins_ui(): void {
 				),
 				array( 'type' => 'error' )
 			);
-			continue;
-		}
-
-		$plugin_data = array_merge(
-			array(
-				'experimental' => false,
-			),
-			$plugin_data, // Data defined within Performance Lab.
-			$api_data
-		);
-
-		// Separate experimental plugins so that they're displayed after non-experimental plugins.
-		if ( $plugin_data['experimental'] ) {
-			$experimental_plugins[ $plugin_slug ] = $plugin_data;
 		} else {
-			$plugins[ $plugin_slug ] = $plugin_data;
+			$plugins[ $plugin_slug ] = array_merge(
+				array(
+					'experimental' => false,
+				),
+				$plugin_data, // Data defined within Performance Lab.
+				$api_data
+			);
 		}
 	}
 
-	if ( count( $plugins ) === 0 && count( $experimental_plugins ) === 0 ) {
+	$plugins = wp_list_sort(
+		$plugins,
+		array(
+			'experimental' => 'ASC',
```

Comment:
This is a bit "too" smart for me, sorting by a boolean is not exactly intuitive. Not opposed to keeping it as is because it _is_ smart, but this kind of thing should have a comment explaining it I think. :)

Reply:
@swissspidy I'll let you amend with an explanatory comment and then you can merge 😄

Reply:
Don‘t really think this warrants a comment, but sure, I can add one tomorrow.